demos/custom-tabs-oauth/build.gradle

```diff
@@ -0,0 +1,42 @@
+/*
+ *    Copyright 2020 Google LLC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.1"
+
+    defaultConfig {
+        applicationId "com.google.browser.examples.custom_tabs_oauth"
```

Comment:
Do you mind making this `com.google.androidbrowserhelper.demos.customtabsoauth`? I'm trying to get all the demos to have packages like that - though I've not got around to a full on refactor yet.

Also happy to standardize on a different package if you've got suggestions.

Could you similarly change the references in the Java code and AndroidManifest.xml?

Reply:
Ha - I did notice we had some variation and went with one we already have. Will change.

Reply:
renamed the id (and package)